app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,47 @@
+class RatingsController < ApplicationController
+  before_action :find_product, :logged_in_user, :has_bought_item?, only: :create
+
+  def create
+    post = Rating.get_post(current_user.id, @product.id).first
```

Comment:
```suggestion
    post =  current_user.ratings.by_product(@product.id).first
```
e gọi thông qua association nhé